Show appropriate messages if an annotation is unavailable to a user.

sheetaluk opened the pull request:
If a selected annotation is unavailable to a user, prompt user to sign in
if he/she isn't signed in.
If a selected annotation is unavailable to a user and he/she is already
signed in, display a message.

Trello:
https://trello.com/c/scNGUMBk/330-improve-copy-to-clarify-direct-linking-functionality

robertknight commented:
The functionality works fine but the text wrapping looks odd:

![3326-message-wrapping](https://cloud.githubusercontent.com/assets/2458/15242013/61fbf61a-18eb-11e6-8690-f6e1b7c3dd22.png)

I'd suggest that either each sentence should appear on its own line or the whole message should appear on one line.

robertknight commented on h/static/styles/thread.scss:
```diff
@@ -20,7 +20,7 @@ $thread-padding: $annotation-card-left-padding;
 
   &__label {
-    max-width: 160px;
+    max-width: 180px;
```
You can't tell from looking at this what the intended visual effect is, that a line break should appear after the end of the sentence. Any reason not to just use `<br>` tags in the template?

finitud commented:
May I suggest to use singular "they" instead of "he/she" for better inclusiveness?

robertknight commented:
> May I suggest to use singular "they" instead of "he/she" for better inclusiveness?

I presume you're referring to the commit message @finitud ?

sheetaluk commented:
1. Changed the commit message.
2. Removed styling for width and added <br>

Here is how it looks:
https://www.dropbox.com/s/dgg1hl61ku3hcrr/Screenshot%202016-05-25%2011.54.18.png?dl=0

nickstenning commented:
LGTM.
